config/bootstrap.php

```diff
@@ -43,27 +42,6 @@
     ]);
 }
 
-Router::plugin('DebugKit', function ($routes) {
```

Comment:
Scopes can't really be overwritten, only appended into.

Have you tried using `Plugin::routes('DebugKit')` to see if routes were enabled? If they are not they can be conditionally loaded.